DEVELOPMENT.md

```diff
@@ -6,8 +6,8 @@
 1.  Create [a GitHub account](https://github.com/join)
 1.  Setup
     [GitHub access via SSH](https://help.github.com/articles/connecting-to-github-with-ssh/)
+1.  Set up your [development environment](#environment-setup)
 1.  [Create and checkout a repo fork](#checkout-your-fork)
-1.  Set up your [shell environment](#environment-setup)
 1.  Install [requirements](#requirements)
```

Comment:
i think you deleted this section so maybe we should remove this also?

Reply:
Ahh good catch, updated.